### ISSUE TITLE
problem with prediction consistency

### ISSUE THREAD

**Issue by eleanor11** [Megvii-BaseDetection/DenseTeacher#3]
Hi, I notice the input of teacher and student model are different (unsup_strong for student and unsup_weak for teacher). And there are some crop and flip operation in data augmentation. 
So will there be some problem with the prediction consistency when directly computing the loss with teacher's ans student's predictions (logits, deltas and quality) ? Or is there any extra transform operations to the predictions in this project?

**Comment by ZRandomize** [comment]
Please notice that the StrongAug is applied on **the results** of WeakAug, and the StrongAug it self contains no geometric transformation. In that case, the student and teacher model should be consistent

**Comment by eleanor11** [comment]
> 

okay，thank you！
